### ISSUE TITLE
Tokei 11 not showing output on Windows 10

### ISSUE THREAD

**Issue by connorskees** [XAMPPRocky/tokei#472]
After updating to version 11, `tokei` no longer prints anything to the console on Windows 10. This appears to be a regression from version `v10.1.0`

To install I did `cargo install tokei --force`, 
```Replaced package `tokei v10.1.0` with `tokei v11.0.0` (executable `tokei.exe`)```
```
tokei --version
tokei 11.0.0 compiled without serialization formats.
```
This issue also occurs with
```
tokei --version
tokei 11.0.0 compiled with serialization support: cbor, json, yaml
```
`tokei` does work for me in Ubuntu 16.04.4 LTS

I wasn't able to find anything immediately interesting in the diff, https://github.com/XAMPPRocky/tokei/compare/v10.1.0...v11.0.0

**Comment by XAMPPRocky** [comment]
@connorskees Thank you for your issue! I'd like to ask if you could first download and try one of the prebuilt binaries on [GitHub][gh], and if you could tell if the problem also exists there I would appreciate it.

I don't have a Windows machine, so I don't have a way to test this. If anyone is willing to debug it I would be happy to help them with providing a fix.

[gh]: https://github.com/XAMPPRocky/tokei/releases/latest

**Comment by connorskees** [comment]
`tokei-x86_64-pc-windows-msvc` does work

In addition, I've cloned master and `cargo r` works fine. 

It seems to be an issue only with the version on `crates.io`. I'd be happy to help debug anything, but it seems that this should be fixed by just bumping it to `v11.0.1`?

**Comment by XAMPPRocky** [comment]
Going to close since it seems like it only affected that version. If you continue to have this issue feel free to re-open.
